modules/oe_content_person/config/install/field.field.oe_person_job.default.oe_role_reference.yml

```diff
@@ -0,0 +1,34 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.oe_person_job.oe_role_reference
+    - oe_content_person.oe_person_job_type.default
+  module:
+    - rdf_skos
+id: oe_person_job.default.oe_role_reference
+field_name: oe_role_reference
+entity_type: oe_person_job
+bundle: default
+label: Role
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:skos_concept'
+  handler_settings:
+    target_bundles: null
+    sort:
+      field: _none
+    auto_create: false
+    concept_schemes:
+      - 'http://publications.europa.eu/resource/authority/corporate-body'
```

Comment:
It has to be http://publications.europa.eu/resource/authority/role. Waiting https://github.com/openeuropa/triple-store-dev/pull/40.

Reply:
we have a new tag 1.10, so changes can be made